sources/lib/reasoners/intervalCalculus.ml

```diff
@@ -1423,6 +1425,42 @@ let rec loop_update_intervals are_eq env cpt =
   then env
   else loop_update_intervals are_eq env cpt
 
+let calc_pow a b ty uf =
+  let ra, expl_a = Uf.find uf a in
+  let rb, expl_b = Uf.find uf b in
+  let pa = poly_of ra in
+  let pb = poly_of rb in
+  match P.is_const pa, P.is_const pb with
+  | Some c_x, Some c_y ->
+    begin
+      match Arith.calc_power_opt c_x c_y ty with
+      | None -> None
+      | Some res -> Some (res, Ex.union expl_a expl_b)
+    end
+  | _ -> None
```

Comment:
What does this function do ? Compute powers of intervals that happen to be single points/constants ?

Reply:
Yes, For now it's only compute if intervals or a constants.